sunpycube/cube/datacube.py

```diff
@@ -75,42 +76,52 @@ def __init__(self, data, wcs, errors=None, **kwargs):
         # Also it's called axes_wcs because wcs belongs to astropy.nddata and
         # that messes up slicing.
 
-    def plot_wavelength_slice(self, offset, axes=None,
-                              style='imshow', **kwargs):
+    def plot_wavelength_slice(self, offset, axes=None, unit_x_axis=None,
+                            unit_y_axis=None, origin='lower', **kwargs):
```

Comment:
We should not have these unit kwargs here, as the user can use the wcaxes api to change them after calling this function. We could consider trying to come up with a better way of defaulting the unit or the formatting later.